common/src/androidTest/kotlin/DatabaseTest.kt

```diff
@@ -0,0 +1,83 @@
+import com.cuhacking.atlas.db.*
+import com.cuhacking.atlas.db.Database.Companion.Schema
+import com.squareup.sqldelight.db.SqlDriver
+import com.squareup.sqldelight.sqlite.driver.JdbcSqliteDriver
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+import kotlin.test.assertTrue
+
+lateinit var driver: SqlDriver
+lateinit var queries: FeatureQueries
+
+class DatabaseTest {
+
+    private val sampleFeature = GeoJsonAdapter.decode("""{"type":"Feature","geometry":{"type":"Point","coordinates":[-75.0,45.0]},"properties":{}}""")
+
+    @Before
+    fun setup() {
+        driver = JdbcSqliteDriver(JdbcSqliteDriver.IN_MEMORY)
+        Schema.create(driver)
+        val database = driver.createDatabase()
+        queries = database.featureQueries
+
+        val feature1 = Feature(1, "River Building","Richcraft Hall","Building",null,null,null,sampleFeature)
+        val feature2 = Feature(2, "RB 2311", null, "room","River Building", "RB2",null,sampleFeature)
+
+        queries.insertFeature(feature1)
+        queries.insertFeature(feature2)
```

Comment:
```suggestion
        database.featureQueries.run {
            insertFeature(feature1)
            insertFeature(feature2)
        }
```

We can use a [scope function](https://kotlinlang.org/docs/reference/scope-functions.html) here.
I'm not sure but we may even be able to add a query that allows multiple features to be inserted at once?

Reply:
I believe there isn't a query that allows the insertion of multiple queries at once, so a scope function would be the best solution like you mentioned!